invenio_saml/views.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-saml is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Blueprint definitions."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint
+
+blueprint = Blueprint(
+    "invenio_saml",
+    __name__,
+    template_folder="templates",
+    static_folder="static",
```

Comment:
In Invenio, we have now supports for theming. Can you add at least the `Semantic-UI` folder for the template?
So that new users will have such theme. I am now wondering if it already uses that theme...